SDALib-CPP/SpeedDreams/tgf/linuxspec.cpp

```diff
@@ -83,367 +83,367 @@ static const size_t SOPathLenMax = 1024;
  *	   already exists in modlist).
  *	
  */
-static int
```

Comment:
This file contains a lot of commented out code, is that intended?

Reply:
not written by me so yes